### PR TITLE
feat(doc-framework): add dark theme toggle for full page examples

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly-org/issues/3055

### CODE REVIEW

File: packages/v4/patternfly-docs/patternfly-docs.config.js
Comment (nicolethoen) [inline packages/v4/patternfly-docs/patternfly-docs.config.js:14]: You committed this - do you mean don't merge this?

Comment (mcoker) [reply packages/v4/patternfly-docs/patternfly-docs.config.js:14]: Oops, yep! When we get an approval on the PR from @mceledonia or @mmenestr, I'll remove that.

**Review by mceledonia — APPROVED** [review]

**Pull request merged**
